src/components/colorPicker/ColorPickerDialog.tsx

```diff
@@ -20,7 +20,6 @@ import Text from '../text';
 import TouchableOpacity from '../touchableOpacity';
 import Dialog, {DialogProps} from '../dialog';
 import Button from '../button';
-//@ts-expect-error
```

Comment:
This shows as an error in TS compile, however I don't see a reason for it, so I removed it.